image/types/src/lib.rs

```diff
@@ -392,4 +392,15 @@ impl ImageTocEntry {
     pub fn image_size(&self) -> u32 {
         self.size
     }
+
+    pub fn overlaps(&self, other: &ImageTocEntry) -> bool {
+        let self_load_addr_end = self.load_addr + self.image_size() - 1;
```

Comment:
You can get rid of `-1` if you replace `<=` with `<` and `>=` with `>`

Reply:
This was done for clarity of the addr. ranges; updated to address the feedback.